csharp/ql/test/library-tests/csharp7/TupleTypes.expected

```diff
@@ -4,6 +4,8 @@
 | (Int32,Double) | (int, double) | ValueTuple<Int32, Double> | 2 | 1 | CSharp7.cs:215:11:215:16 | Item2 |
 | (Int32,Int32) | (int, int) | ValueTuple<Int32, Int32> | 2 | 0 | CSharp7.cs:64:10:64:10 | Item1 |
 | (Int32,Int32) | (int, int) | ValueTuple<Int32, Int32> | 2 | 1 | file://:0:0:0:0 | Item2 |
+| (Int32,String) | (int, string) | ValueTuple<Int32, String> | 2 | 0 | CSharp7.cs:97:19:97:19 | Item1 |
+| (Int32,String) | (int, string) | ValueTuple<Int32, String> | 2 | 1 | CSharp7.cs:97:22:97:37 | Item2 |
 | (String,Int32) | (string, int) | ValueTuple<String, Int32> | 2 | 0 | CSharp7.cs:84:17:84:17 | Item1 |
 | (String,Int32) | (string, int) | ValueTuple<String, Int32> | 2 | 1 | file://:0:0:0:0 | Item2 |
```

Comment:
@hvitved How do we come up with the locations of these items? The unmodified line 9 and 10 looks particularly strange here: `Item1` and `Item2` of the same tuple type shows up with different locations.

Reply:
Good question: `CSharp7.cs:84` contains `return (a: x, 2).a;`, so perhaps we only get locations for explicitly named components.